extensions/spatial.sigmf-ext.md

```diff
@@ -0,0 +1,362 @@
+# The `spatial` SigMF Extension Namespace v0.0.2
+
+This document defines the `spatial` extension namespace for the Signal Metadata
+Format (SigMF) specification. This extension namespace contains objects to help
+store information about spatially diverse data, specifically information applied
+to directional antennas and multichannel phase-coherent datasets used for signal
+direction of arrival and beamforming.
+
+Multichannel datasets can be stored in SigMF Collections or as multichannel
+interleaved data depending on the application, Collections are RECOMMENDED due
+to the easier processing and better application support.
+
+The `spatial` extension makes use of cartesian coordinates to define array
+geometry, and spherical coordinates for reporting bearings. This, coupled with
+the various methods for defining boresights and bearing references can become
+complicated rapidly so the union of these two cordinate reference systems (CRS)
+is illustrated in Figure 1 below. The ISO 80000-2:2019 compliant cartesian and
+spherical systems have a specific relationship with the reported azimuth and
+elevation, which use the conventional geospatial definitions (degrees east of
+north, and degrees above horizon respectively).
+
+![SigMF Spatial Coordinate Reference System](spatial_crs.png)
+
+**Figure 1 - SigMF Spatial Extension Coordinate Reference Systems**
+
+As shown in Figure 1, the boresight of the aperture is defined as being in the
```

Comment:
Is this a rule or a happenstance of Figure 1?

If a rule, what benefit does this rule (boresight must be the x-axis) bring, other than making it obvious which direction boresight is? Shouldn't that be explicit instead? Boresight is well defined for many array geometries, so this becomes impossible to comply with. I think the antenna geometry *description* should be independent of the array response.

Reply:
Good discussion, here. I'm on-board with the proposed approach, @jacobagilbert 👍🏼